larq_compute_engine/tflite/tests/bconv2d_test.cc

```diff
@@ -416,20 +429,24 @@ TEST_P(BConv2DOpTest, SimpleTest) {
   std::generate(std::begin(input_data), std::end(input_data), rand_generator);
   std::generate(std::begin(filters_data), std::end(filters_data),
                 rand_generator);
-  std::generate(std::begin(channel_multipliers), std::end(channel_multipliers),
-                float_generator);
-  std::generate(std::begin(bias_data), std::end(bias_data), float_generator);
+  std::generate(std::begin(post_activation_multiplier_data),
+                std::end(post_activation_multiplier_data), float_generator);
```

Comment:
why `post_activation_multiplier` can only be these numbers in float_list{0.125, 0.25, 0.75, 0.875, 1.0}? why not create them randomly instead of randomly choosing one of these?

Reply:
Thanks for spotting this. This is mainly a remnant of the fact that we first fused it with the backtransform. Since the order of multiplications is different than in tflite, some numbers produced float-precision errors, but these nice "round" numbers (at least in base 2) do not.

I will change it to random floats in the [-1,1] interval.